generic/tests/netperf.py

```diff
@@ -409,46 +409,46 @@ def start_test(server, server_ctl, host, clients, resultsdir, test_duration=60,
                 ret = launch_client(j, server, server_ctl, host, clients,
                                     test_duration, nf_args, netserver_port,
                                     params, server_cyg, test)
-
-                thu = float(ret['thu'])
-                cpu = 100 - float(ret['mpstat'].split()[mpstat_index])
-                normal = thu / cpu
-                if ret.get('tx_pkt') and ret.get('exits'):
-                    ret['tpkt_per_exit'] = float
-                    (ret['tx_pkts']) / float(ret['exits'])
-
-                ret['size'] = int(i)
-                ret['sessions'] = int(j)
-                if protocol in ("TCP_RR", "TCP_CRR"):
-                    ret['trans.rate'] = thu
+                if ret:
```

Comment:
What's the vaule of ret? I didn't see any return value of function launch_client.

Reply:
return value is "return ret" in line 713